app-dicts/myspell-en/metadata.xml

```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<maintainer type="person">
+		<email>onigino@protonmail.com</email>
+		<name>Gino McCarty</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>English dictionaries for OpenOffice/LibreOffice 4.0+ in an individual extension.</longdescription>
```

Comment:
What's the purpose of having a **long** description that's no longer from DESCRIPTION?